java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -310,6 +310,37 @@ public BaseDeviceMemoryBuffer getDeviceBufferFor(BufferType type) {
     return srcBuffer;
   }
 
+  /**
+   * Create a ColumnVector from the off heap Apache Arrow buffers passed in.
+   * @param type - type of the column
+   * @param colName - Name of the column
```

Comment:
It's odd to see a column name in this API since cudf doesn't use column names anywhere else.

Reply:
true, I'll hardcode it as it doesn't matter on the cudf side